btree/btree.c

```diff
@@ -2204,6 +2220,10 @@ static void bnode_put(struct node_op *op, struct nd *node)
 		 */
 		ndlist_tlist_del(node);
 		ndlist_tlist_add(&btree_lru_nds, node);
+		#ifndef __KERNEL__
+		to_purge  = lru_slow_purge ?
+			   ((m0_be_chunk_header_size() + node->n_size) * 2) : 0;
```

Comment:
s/2/TRICKLE_MULTIPLIER/
Add this #define TRICKLE_MULTIPLIER 2 so that we can modify this macro in future as needed.

Reply:
Done.